examples/machine-learning-pipeline/README.md

```diff
@@ -420,519 +672,1534 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to get
    cd ..
    ```
 
-## Running Terraform locally
+### VPC-SC - Infrastructure Deployment with Local Terraform - Only proceed with these if you have not used Cloud Build
```

Comment:
Move the section `VPC-SC - Infrastructure Deployment with Local Terraform - Only proceed with these if you have not used Cloud Build` as a subsection to `Prerequisites` after the VPC-SC instructions for cloudbuild

Reply:
and remove the duplicated "Usage" sections that is right above it

Reply:
I created [this issue ](https://github.com/GoogleCloudPlatform/terraform-google-enterprise-genai/issues/68)as an improvement. I believe we having two separated READMEs will keep the doc more organized.